test/meson.build

```diff
@@ -26,3 +26,8 @@ if get_option('enable-capi')
   subdir('tizen_capi')
   subdir('unittest')
 endif
+
+if get_option('enable-nnstreamer-tensor-filter')
+  install_data('nnstreamer_filter_nntrainer/nnstreamer-test.ini', install_dir: application_install_dir)
```

Comment:
Do we have to install this? Cause nnstreamer have it already.

Reply:
removed this as per discussion offline.